notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -2,7 +2,7 @@
   "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model_display_name = f'{"e2e-text-classification-model-unique"}'
</pre><p>Remove curly braces and f.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>